test/e2e/operator_test.go

```diff
@@ -364,6 +380,59 @@ func customResolver(nameserver string) *net.Resolver {
 	}
 }
 
+func ensureOperandNamespace() error {
+	err := kubeClient.Create(context.TODO(), &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "external-dns"}})
+	return err
```

Comment:
```suggestion
	return kubeClient.Create(context.TODO(), &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "external-dns"}})
	return err
```
Can you also make a constant for the operand namespace?

Reply:
Done!